Update setup and running instructions

andrewmogan opened the pull request:
The current README is essentially a list of commands with little context or explanation. There's also no explanation of ssh tunneling, which may not be obvious to everyone. These README updates attempt to clarify these points.

andrewmogan committed:
Update setup and running instructions

alessandrothea commented on README.md:
```diff
@@ -1,25 +1,31 @@
+git clone https://github.com/DUNE-DAQ/justintime.git
 cd justintime
 pip install -r requirements.txt
+source env.sh
```
This is needed for running just in time, not for the setup.
Remove

alessandrothea commented on README.md:
````diff
@@ -1,25 +1,31 @@
-cd <workarea>
+To use `justintime`, you should first setup up a DUNE DAQ environment. You can follow the instructions in the [DUNE DAQ wiki](https://dune-daq-sw.readthedocs.io/en/latest/packages/daq-buildtools/) to do so. For convenience, the steps below will set up a local environment based on a nightly build of DUNE DAQ. These steps are tested and working as of March 2024, but in case they become outdated, pl ...
+```
+source /cvmfs/dunedaq.opensciencegrid.org/setup_dunedaq.sh
````
Tis is a repeat of basic daq area instructions and should be avoided. 
A link to basic setup instructions should be used instead.

alessandrothea commented on README.md:
```diff
@@ -1,25 +1,31 @@
-cd justintime
-source env.sh
-
+To run `justintime`, you need a directory containing HDF5-format DUNE DAQ data files and you need to select a channel map. As of March 2024, the available channel map options are 'VDColdbox', 'ProtoDUNESP1', 'PD2HD', 'VST', 'FiftyL', and 'ICEBERG'. 
```
Why did you remove the environment setup commands?

andrewmogan committed:
Implement comments

- Better separation between setup and running instructions (i.e., put `source justintime/env.sh` under the running section)
- Remove redundant dev area setup instructions
